edb/schema/functions.py

```diff
@@ -1445,7 +1449,66 @@ def compile_function(
                     context=body.qlast.context,
                 )
 
-        return_type = self._get_attribute_value(schema, context, 'return_type')
+        return expr
+
+    @classmethod
+    def compile_function(
```

Comment:
I spent a long while being confused about the new `compile_function`/`compile_this_function` before I realized that `compile_function` was a `@classmethod`. I guess a comment could be added but to be honest it's pretty clear once I realized that.

Reply:
I think it'll be even clearer if we turn it into a function, it doesn't need to be a classmethod